packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js

```diff
@@ -1,22 +1,28 @@
 import PropTypes from 'prop-types'
 import React, {useEffect} from 'react'
-import SplitPane from 'react-split-pane'
-import styled from 'styled-components'
+import ReactDOM from 'react-dom'
 import {notification} from 'tocco-app-extensions'
-import {scale} from 'tocco-ui'
+import {Icon} from 'tocco-ui'
 
 import SchedulerAppContainer from '../../containers/SchedulerAppContainer'
 import SearchPanel from '../SearchPanel/SearchPanel'
-import {resizerStyle, StyledSplitPanelWrapperLeft, StyledSplitPanelWrapperRight} from './StyledResourceScheduler'
+import {
+  StyledSplitPanelWrapperLeft,
+  StyledSplitPanelWrapperRight,
+  StyledSplitPane,
+  StyledGutter
+} from './StyledComponents'
 
-const StyledSplitPane = styled(SplitPane)`
-  position: static !important;
-
-  .react-bs-container-body {
-    height: auto !important;
-    margin-bottom: ${scale.space(0)};
-  }
-`
+const getGutter = () => () => {
+  const gutterEl = document.createElement('div')
+  ReactDOM.render(
+    <StyledGutter tabIndex={0}>
```

Comment:
I'm surprised that this is working. I would've assumed that there is no `Theme` available for `StyledGutter` since we are creating a complete new react context with `render`

Reply:
But I have no better idea so far.. I guess when it's working as expected it should be ok for now